ONSdigital/ras-frontstage
# Add session expire-at endpoint

# What and why?
This PR adds the expire-at end point to allow the timeout modal to work. This is more a stepping stone PR as timeout is in 2 stories and without delivering both but knowing what the 2nd part needs it has been developed with that in mind
# How to test?
Deploy this PR to your environment and then hit the session/expires-at endpoint (N.B the time should be displayed as UTC) It shouldn't change if you refresh the page, but as you wander around the rest of frontstage it will
# Trello
